### DISCUSSION TITLE
學生用AI聊天機器人寫作業 教授嘆：難找到代寫鐵證

### DISCUSSION THREAD

**Discussion by skyling0299** [1101-GS3073-C/Python-Community-GS3073-C#548] [General]
一名學生在他的哲學課交了一篇由ChatGPT完成的論文作業，他坦承學術界沒有預見到這一點，讓學者們有點措手不及。

希克透露，這份代寫的報告有一些特徵，看得出是由AI代寫而成。他也表示，要證明AI代筆論文幾乎是不可能，只能從一些蛛絲馬跡推敲。他貼上可疑的文字段落到AI頁面，得到99.9%匹配準確度，他也向ChatGPT詢問一系列該學生可能提問的問題，沒想到也得到了相似答案。

學生如何利用chatGPT交出一份看不出代寫痕跡的作業與教授如何破解學生是否使用GPT生成文章成為難題

資料來源: https://news.ltn.com.tw/news/world/breakingnews/4166507

**Comment by brian431** [comment]
看來以後會有很多隨堂抽考來揪出那些用AI寫報告的學生

**Comment by awkward-willy** [comment]
先前有聽說過其他學校的教授直接開放學生使用 ChatGPT ，前提是在作業中需有明確標示。或許這樣直接開放並鼓勵學生遵守規範的作法比單純強硬的禁止以及抓細節來得更有效果與意義。

**Comment by Ben6uo** [comment]
使用 AI 聊天機器人寫作業可能有一些優點，例如可以提供自動的拼寫和文法檢查，我覺得不錯

**Comment by williamyen0623** [comment]
我放假都在跟chatGPT聊天，他真的可以解決一堆我google都很難找到的問題，有時候也會問他一些奇怪的問題，但他都會文鄒鄒的回應哈哈

**Comment by HsuChen88** [comment]
chatGPT知道好多事，連問她怎麼找到另一半都知道。

**Comment by gnsJhenJie** [comment]
感覺Turnitin這種抄襲檢測平台應該會推出針對檢測文本是否為AI產生的解決方案

**Comment by bww0401** [comment]
沒想到AI機器人連論文都可以幫忙寫了，看來以後會有更多想不到的事情發生: )

**Comment by larry40305** [comment]
用AI寫論文，再用AI偵測AI寫的論文，然後在更新成AI偵測不到的AI寫的論文。
